recipe/meta.yaml

```diff
@@ -20,25 +32,33 @@ outputs:
       run_exports:
         strong:
           - cudatoolkit {{ cuda_compiler_version }}|{{ cuda_compiler_version }}.*
-          - __glibc >=2.17  # [linux and cuda_compiler_version == "11.0"]
+{% if cuda_major_minor >= (11, 0) %}
+          - __glibc >=2.17        # [linux]
+{% endif %}
     requirements:
       build:      # [win]
         - m2-sed  # [win]
       host:
         # Needed to symlink libcuda into sysroot libs.
         - {{ compiler("c") }}
-        - sysroot_linux-64 2.17  # [linux and cuda_compiler_version == "11.0"]
-        - __glibc >=2.17         # [linux and cuda_compiler_version == "11.0"]
+{% if cuda_major_minor >= (11, 0) %}
+        - sysroot_linux-64 2.17   # [linux]
```

Comment:
I thought this should be in the `build` section, not `host`?
https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=sysroot#using-centos-7